crates/primitives/src/header.rs

```diff
@@ -284,9 +284,9 @@ impl Decodable for Header {
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
 pub struct SealedHeader {
     /// Locked Header fields.
-    header: Header,
+    pub header: Header,
     /// Locked Header hash.
-    hash: BlockHash,
+    pub hash: BlockHash,
```

Comment:
Why mark this as pub? Can use `.hash()`

Reply:
I needed to access it for a test, I am not sure if it is the proper way.